src/App.js

```diff
@@ -21,19 +23,166 @@ const firebaseConfig = {
+                    </div>
+                </div>
+            </div>
+        ),
+        "signUp": (
+            <div className={"home-page"}>
+                <img tabIndex="0" onKeyPress={(event) => {(event.key === "Enter"||event.code === "Space") && setPage({type: "trueHome"})}} onClick={() => setPage({type: "trueHome"})} alt={"Back Arrow"} src={"long-arrow-alt-left-solid.svg"} className={"back-arrow"}/>
+                <h1 className={"sign-in-up-h1s"}>Create an Account</h1>
+                {!signupValid && <p className={"wrongPass"}>Invalid email or password</p>}
+                <div className={"buttons-box"}>
+                    <div className={"authbuttons"}>
+                        <input className={"authitems"} placeholder={"Email"} type="text" name="email" onChange={(e) => setEmail(e.target.value)}/>
+                        <input className="authitems" placeholder={"Password"} type="password" name="password" onChange={(e) => setPassword(e.target.value)}/>
+                        <button className="front-page-buttons" onClick={() => handleSignUp(userEmail, userPassword)}> Sign Up </button>
+                        <p className={"authOption"}>or</p>
+                        <button className="front-page-buttons" onClick={() => handleGoogleSignIn()}>Sign Up with Google</button>
+                    </div>
+                </div>
+            </div>
+        ),
+        "trueHome": (
+            <div className={"home-page"}>
+                <h1 className={"homeName"}>LISTIES</h1>
+                <p className={"subtitle"}>By Lucas and Lauren</p>
+                <div className={"buttons-box"}>
+                    <div className={"authbuttons"}>
+                        <button className={"front-page-buttons"} onClick={() => setPage({type: "signIn"})}>Sign In</button>
+                        <p className={"authOption"}>or</p>
+                        <button className={"front-page-buttons"} onClick={() => setPage({type: "signUp"})}>Create an Account</button>
+                    </div>`
+                </div>
+            </div>
+
+        ),
+        "home": (
+            <div>
+                <SignedInPage {...props} user={user} signOut={() => handleSignOut()}/>
+            </div>
+        )
+    }
+    return pageRenderLookup[selectedPage.type]
+}
+
+
+function SignedInPage(props) {
+
+    const collectionName = "App_Data";
+    const query = db.collection(collectionName).where("sharedWith", "array-contains", props.user.email);
     const [hasFetchedTask, setFetch] = useState(false);
-    let collectionRef = db.collection(collectionName)
+    let collectionRef = db.collection(collectionName);
     const [value, loading, error] = useCollection(query)
     if (!loading) {
-        value.docs.map(async (x) => console.log((await getDoc(x.ref)).data().title))
+        if (value){
+            value.docs.map(async (x) => console.log((await getDoc(x.ref)).data().title))
```

Comment:
This is useless call since you are doing nothing with the result of the map.  If you want the side effect of the console.log, use value.docs.foreach (less confusing to reader).  However, no need to have this, just log in line 84.